snapwell/snapwell_main.py

```diff
@@ -103,14 +105,14 @@ def run_and_write(self, wp, wp_date):
 
     def main_loop(self):
         num_snaps = len(self.wellpaths)
-        logging.info("delta_z    = %.3f", self.config.deltaZ())
-        logging.info("owc_offset = %.3f", self.config.owcOffset())
-        owc_def = self.config.owcDefinition()
-        logging.info("owc_defini = %.3f (%s)", owc_def[1], owc_def[0])
-        logging.info("output     = %s", self.config.output())
+        logging.info("delta_z    = %.3f", self.config.delta_z)
+        logging.info("owc_offset = %.3f", self.config.owc_offset)
+        owc_def = self.config.owc_definition
+        logging.info("owc_defini = %.3f (%s)", owc_def.value, owc_def.keyword)
+        logging.info("output     = %s", self.config.output_dir)
         success = True
         for i, wp in enumerate(self.wellpaths):
-            wp_date = self.config.date(i)
+            wp_date = self.config.wellpath_files[i].date
```

Comment:
I guess these properties have to be in sync, but this seems a bit error prone? A `zip` should also work though?

Reply:
Ended up fixing this by adding date to the wellpath object.